mk/docs.mk

```diff
@@ -126,6 +126,7 @@ doc/not_found.html: $(D)/not_found.md $(HTML_DEPS) | doc/
 		--markdown-css https://doc.rust-lang.org/rust.css $<
 
 define DEF_DOC
+RUSTDOC_DEPS_$(1) ?=
```

Comment:
I think for now I'd prefer to avoid these changes. I'm worried about the sprinkling around and I suspect that different platforms will have different sets of warnings (encouraging more sprinklings), and there's nothing making sure that we actually have the "right set of sprinklings". In general this seems like it may end up just growing to be messy over time :(